### ISSUE TITLE
Compiling Error: Packet Handler / Protocolgame.ccp

### ISSUE THREAD

**Issue by soyfabi** [otland/forgottenserver#4417]
Greetings, I am trying to add the: Packet Handler
https://github.com/otland/forgottenserver/pull/4302/files#diff-eb01d79b279ebf659ed73216c0b3c490685221c9358e058ce55a1c8ab4e99c35R852

When compiling I get an error:
![Screenshot_1](https://user-images.githubusercontent.com/91706615/233756315-3c34aae3-8cb4-4a56-bcf0-51ba2fd3ecc4.png)

I don't know how I should place it, since TFS 1.5 nekiro brings it in another way
since it uses the variable: AddGameTask
```
void addGameTask(Callable&& function, Args&&... args) {
   g_dispatcher.addTask(createTask(std::bind(std::forward<Callable>(function), &g_game, std::forward<Args>(args)...)));
}
```

```
In short, how should I place this line:
g_dispatcher.addTask([=, playerID = player->getID(), msg = new NetworkMessage(msg)]() {
				g_game.parsePlayerNetworkMessage(playerID, recvbyte, msg);
});
```
This is my protocolgame.ccp
https://github.com/soyfabi/Forgotten-Server/blob/main/src/protocolgame.cpp#L551

**Comment by EPuncker** [comment]
https://otland.net/forums/support.16/

this issue is not about tfs so should be dealt in forum
